src/configurations/destinations/tune/db-config.json

```diff
@@ -33,7 +33,7 @@
       "warehouse": ["cloud"]
     },
     "destConfig": {
-      "defaultConfig": ["tuneEvents"],
+      "defaultConfig": ["subdomain", "tuneEvents"],
```

Comment:
What about tuneEvents configuration field ?

Reply:
@krishna2020 here `tuneEvents` is the `configKey` for the whole dynamicCustomForm. So, other row fields are also getting included.